Mathlib/Algebra/Order/AbsoluteValue/Basic.lean

```diff
@@ -373,6 +384,58 @@ lemma IsNontrivial.exists_abv_lt_one (h : v.IsNontrivial) : ∃ x ≠ 0, v x < 1
   rw [map_inv₀]
   exact (inv_lt_one₀ <| v.pos hy₀).mpr hy
 
+theorem inv_pos {x : R} (h : 0 < v x) : 0 < v x⁻¹ := by
+  rwa [map_inv₀, _root_.inv_pos]
+
+omit [ExistsAddOfLE S] in
+theorem ne_zero_of_one_lt {x : R} (hv : 1 < v x) : x ≠ 0 := by
+  contrapose! hv
+  simp [hv]
+
+theorem isNontrivial_iff_exists_abv_one_lt :
+    v.IsNontrivial ↔ ∃ x, 1 < v x := by
+  refine ⟨fun h => h.exists_abv_gt_one, fun ⟨x, hx⟩ => ⟨x⁻¹, ?_, ?_⟩⟩
+  · simpa only [ne_eq, inv_eq_zero] using ne_zero_of_one_lt hx
+  · simpa only [map_inv₀, ne_eq, inv_eq_one] using ne_of_gt hx
+
+omit [ExistsAddOfLE S] [IsStrictOrderedRing S] in
+theorem nonpos_iff {x : R} : v x ≤ 0 ↔ v x = 0 := by
+  simp [le_antisymm_iff, v.nonneg _]
+
+theorem inv_lt_one_iff {x : R} : v x⁻¹ < 1 ↔ x = 0 ∨ 1 < v x := by
+  simp only [map_inv₀, inv_lt_one_iff₀, nonpos_iff, map_eq_zero]
+
+theorem mul_one_div_lt_iff {y : R} (x : R) (h : 0 < v y) : v (x * (1 / y)) < 1 ↔ v x < v y := by
```

Comment:
I do not understand this naming, but more importantly I feel that this and the following lemma are too specialized (and with an easy `rw` proof) to deserve to be added to the library.